draft-ietf-quic-tls.md

```diff
@@ -947,7 +947,7 @@ from the Destination Connection ID field from the client's first Initial
 packet.
 
 This secret is determined by using HKDF-Extract (see Section 2.2 of
-{{!HKDF=RFC5869}}) with a salt of 0xafbfec289993d24c9e9786f19c6111e04390a899
+{{!HKDF=RFC5869}}) with a salt of 0x38762cf7f55934b34d179ae6a4c80cadccbb7f0a
```

Comment:
Out of curiosity, how were these new salts derived? Are we going to learn after publication that this was a hash of @martinthomson 's social security number?

Reply:
My guess is that he rolled some D&D dice.

Reply:
Revealing how it was done would ruin the mystery.

Reply:
I mean, I was sort of hoping for us to use a provable process here à la NomCom selection (pick a future date and hash some unpredictable values like weather in San Francisco and the AAPL stock price on that day) to ensure that @martinthomson doesn't accept money from AnOnyMoUs so they don't hack all our initial packets.

Reply:
Hacking initial packets doesn't need a lot of mystery-solving -- the salt is published here :-)

Reply:
@janaiyengar yes that was a joke :-) but more seriously I was truly expecting us to come up with a convoluted way to establish this salt for the final version to ensure it is impartial. Not that I'll lie down on the tracks over this, as a wise person one said...